cmd/frpc/sub/root.go

```diff
@@ -74,6 +74,7 @@ var (
 	bindPort          int
 
 	tlsEnable bool
+	tls_trusted_ca_file string
```

Comment:
Adds a parameter `tls_trusted_ca_file` to the FPRC binary that allows passing in a path to a root certificate file